### PR TITLE
Fixes a random invisible wall

### PR DESCRIPTION
**Pull request by Bram-KEP**
## About The Pull Request
Fixes a random invisible wall.
image

## Why It's Good For The Game
Bugs bad

### CODE REVIEW

**Comment by Bram-KEP** [comment]
the screenshot is fucked for some reason, don't worry about it

File: maps/map_files/USS_Almayer/USS_Almayer.dmm [hunk 1]
```diff
@@ -24070,9 +24070,7 @@
 	},
 /area/almayer/living/cryo_cells)
 "bAZ" = (
-/obj/structure/machinery/cm_vending/clothing/vehicle_crew{
-	pixel_x = 32
-	},
+/turf/open/space/basic,
```
Comment (Nanu308) [inline maps/map_files/USS_Almayer/USS_Almayer.dmm:24073]: space? What you are meant to do is move the vendor in by pixel shifting it East and changing the density similar to the other vendors. There is still meant to be a wall below it.

**Review by Nanu308 — CHANGES_REQUESTED** [review]

File: maps/map_files/USS_Almayer/USS_Almayer.dmm (hunk 1 above)
Comment (Bram-KEP) [reply maps/map_files/USS_Almayer/USS_Almayer.dmm:24073]: Yeah I think I messed something up, will fix

File: maps/map_files/USS_Almayer/USS_Almayer.dmm [hunk 2]
```diff
@@ -23972,9 +23972,7 @@
...
```
Comment (stanalbatross) [inline maps/map_files/USS_Almayer/USS_Almayer.dmm:23975]: what's going on here? it seems to be the exact same as when nanu remarked on it

**Review by stanalbatross — CHANGES_REQUESTED** [review]

File: maps/map_files/USS_Almayer/USS_Almayer.dmm (hunk 2 above)
Comment (Bram-KEP) [reply maps/map_files/USS_Almayer/USS_Almayer.dmm:23975]: I have no idea, thought I offset it

**Pull request closed without merging**